app/views/graphiql/rails/editors/show.html.erb

```diff
@@ -4,6 +4,9 @@
     <title>GraphiQL</title>
     <%= stylesheet_link_tag("graphiql/rails/application") %>
     <%= javascript_include_tag("graphiql/rails/application") %>
+    <% if GraphiQL::Rails.config.csrf %>
+    <%= csrf_meta_tags %>
```

Comment:
Didn't realize at first that `form_authenticity_token` will use it's own token. Changed to use javascript to get the meta tag with the token.